gap/Magic.gd

```diff
@@ -2,7 +2,7 @@
 ##
 ##  AutoDoc package
 ##
-##  Copyright 2012-2016
+##  Copyright 2012-2019
```

Comment:
Please remove this from this PR, it doesn't belong in there (and it's conflicting with PR #210, which makes this change superfluous).